pom.xml

```diff
@@ -530,6 +519,29 @@
         </plugins>
       </build>
     </profile>
-
+    <profile>
+      <id>apache-release</id>
+      <build>
+        <plugins>
+          <plugin>
+            <artifactId>maven-assembly-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>source-release-assembly</id>
+                <configuration>
+                  <!-- make the source release zip includes the word incubating -->
+                  <finalName>apache-zipkin-dependencies-incubating-${project.version}</finalName>
```

Comment:
I forgot to paste this section and noticed when doing a trial release that the name convention was wrong and also some spurious files were in the zip